Formula/weather.rb

```diff
@@ -0,0 +1,26 @@
+class Weather < Formula
+  desc "Command-line weather"
```

Comment:
maybe a few more words

Reply:
`Command-line weather from the darksky.net API`?